content/en/docs/concepts/overview/components.md

```diff
@@ -53,8 +53,8 @@ Some types of these controllers are:
   * Node controller: Responsible for noticing and responding when nodes go down.
   * Job controller: Watches for Job objects that represent one-off tasks, then creates
     Pods to run those tasks to completion.
-  * Endpoints controller: Populates the Endpoints object (that is, joins Services & Pods).
-  * Service Account & Token controllers: Create default accounts and API access tokens for new namespaces.
+  * Endpoint slice controller: Populates EndpointSlice objects (to provide a link between Services and Pods).
```

Comment:
nit: use code formatting for API object names

Reply:
I think I'm agreeing with the comment above, I always prefer "EndpointSlice controller"

Reply:
```suggestion
* EndpointSlice controller: Populates EndpointSlice objects (to provide a link between Services and Pods).
```